alpaca/rest.go

```diff
@@ -471,21 +498,27 @@ func (c *Client) GetTrades(
 			if tradeResp.NextPageToken == nil {
 				return
 			}
-			pageToken = *tradeResp.NextPageToken
-			total += len(tradeResp.Trades)
+			q.Set("page_token", *tradeResp.NextPageToken)
+			received += len(tradeResp.Trades)
 		}
 	}()
 
 	return ch
 }
 
-// GetQuotes returns a channel that will be populated with the quotes for the given symbol
-// that happened between the given start and end times, limited to the given limit.
+// GetQuotesParams contains optional parameters for getting quotes
+
```

Comment:
similar comments as with `GetTradesParams`
🤔 would it make sense to create a struct that contains common parameters and embed it? what do you think?

Reply:
The reason I didn't do it (and wouldn't like to) is because from the user's perspective it's a little bit easier to create this struct:
```go
alpaca.GetTrades(symbol, GetTradesParams{Start: start})
```
than this
```go
alpaca.GetTrades(symbol, GetTradesParams{Interval: IntarvalParams{Start: start}})
```